### ISSUE TITLE
[a11y]: Tear sheet hidden-visually reported as tabable by a11y

### ISSUE THREAD

**Issue by MitchellCalder** [carbon-design-system/ibm-products#8669]
### Package

@carbon/ibm-products

### Browser

Chrome

### Operating System

MacOS

### Package version

2.75.0

### Automated testing tool and ruleset

AxeBuilder a11y

### Description

Accessibility scan for our product ran into an issue where it seems that there is a visually hidden element that is getting caught as a tabable field, but since it is visually hidden it is lacking any accessibility fields like label that a reader would be looking for. I'll include details in the thread.

Target: 
`["#bus-iframe",["c4p-tearsheet","#start-sentinel"]]`

HtmlElement: 
`<a id="start-sentinel" href="javascript:void 0" role="navigation" class="c4p--visually-hidden"></a>`

and

Target:
`["#bus-iframe",["c4p-tearsheet","#end-sentinel"]]`

HtmlElement:
`<a id="end-sentinel" href="javascript:void 0" role="navigation" class="c4p--visually-hidden"></a>`

<img width="3452" height="1444" alt="Image" src="https://github.com/user-attachments/assets/950ad502-acf9-44a2-b4b5-b2594d9db7f3" />

The a11y tool has it flagged under the following:
```
"Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute"
```

### Reproduction/example

Open https://ibm-products-web-components.carbondesignsystem.com/?path=/story/components-tearsheet--with-influencer and run accessibility check.

### Steps to reproduce

Open https://ibm-products-web-components.carbondesignsystem.com/?path=/story/components-tearsheet--with-influencer and run accessibility check.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems

**Comment by nrbarry** [comment]
Hi team, I am curious if there is any projection as to if/when this might get prioritized by the carbon team?  We are trying to push a solution to production and this is impacting our accessibility signoff.
